routes/middleware/post_place.js

```diff
@@ -1,38 +1,30 @@
-const request = require('request');
+const axios = require('axios');
 require('dotenv').config();
 
-module.exports = (req, res, next) => {
-  var url = `${process.env.URI}/places`;
+module.exports = async ({ body }, res, next) => {
```

Comment:
You could maybe go a step further and get the properties manually: {body: {place_name_en, place_name_ar}}

Reply:
Even if there are more you could add them at the top if you wanted to. Completely up to you, but it would save a bit of code underneath :man_shrugging:

Reply:
there is 4 more that are going to be added so that is 6 var names in the post_place handler arguments, it just going to be really messy